tf2_ros/src/tf2_ros/buffer.py

```diff
@@ -67,7 +67,10 @@ def __init__(self, cache_time = None):
         self._callbacks_to_remove = []
         self._callbacks_lock = threading.RLock()
 
-    def __get_frames(self, req):
+        if debug:
+            self.srv = node.create_service(FrameGraph, 'tf2_frames', self.__get_frames)
+
+    def __get_frames(self, req, res):
```

Comment:
This looks odd. The function gets a `res` argument but returns the response?

Reply:
I'm using the same idea as in the ROS 2.0 [demos](https://github.com/ros2/demos/blob/master/demo_nodes_py/demo_nodes_py/services/add_two_ints_server.py#L27) or [examples](https://github.com/ros2/examples/blob/master/rclpy/services/minimal_service/examples_rclpy_minimal_service/service_member_function.py#L27). is it not need it to return the response?

Reply:
I created a separate ticket for this (ros2/rclpy#464) so the patch can stay as is.